lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -698,6 +699,11 @@ function dosomething_user_user_insert(&$edit, &$account, $category = NULL) {
     $edit['name'] = $account->uid;
     $account->name = $account->uid;
   }
+
+  // Trigger an analytics event on registration
+  // @NOTE: The 'Authentication - Login' event will *also* be triggered here,
+  // since registered users are subsequently logged in to their new accounts.
+  dosomething_helpers_add_analytics_event('Authentication', 'Register');
```

Comment:
instead of doing this in `hook_insert()` maybe we can do it in the `new_user` submit handler it was in before. it's not a huge deal, but since it's not changing the user object, and it doesn't need to be in the insert hook. just a thought.